pyproject.toml

```
@@ -59,7 +59,7 @@ xarray = ">=22.6.0"
 tabulate = ">=0.8.10"
 jinja2 = ">=3.1"
 scipy = { version = ">=1.9.1", python = "<3.14" }
-SQLAlchemy = ">=2.0.12,<2.0.39"
+SQLAlchemy = ">2.0.39"
```

```suggestion
SQLAlchemy = ">=2.0.39"
```

you missed the `=` sign.